791 add fields to fieldofview in session

arielleleon opened the pull request:
Added new fields to the FieldOfView session object required for multiplane sessions.

jtyoung84 commented on src/aind_data_schema/core/session.py:
```diff
@@ -73,6 +77,14 @@ class FieldOfView(AindModel):
     frame_rate_unit: FrequencyUnit = Field(FrequencyUnit.HZ, title="Frame rate unit")
     coupled_fov_index: Optional[int] = Field(None, title="Coupled FOV", description="Coupled planes for multiscope")
+    power: Optional[Decimal] = Field(None, title="Power (mW)")
+    power_unit: Optional[PowerUnit] = Field(PowerUnit.MW, title="Power unit")
```
I think we've been keeping the Unit fields as non-Optional and set with a default

jtyoung84 commented on src/aind_data_schema/core/session.py:
```diff
@@ -73,6 +77,14 @@ class FieldOfView(AindModel):
+        None,
+        title="Z stage position of the fastz actuator for a given targeted depth",
+    )
+    scanfield_z_unit: Optional[SizeUnit] = Field(None, title="Z stage position unit")
```
I think we've been keeping the Unit fields as non-Optional and set with a default

arielleleon replied:
@jtyoung84  Got it and fixed

jtyoung84 reviewed (changes requested):
The examples in the examples folder may need to be updated

arielleleon commented:
@jtyoung84 Examples have been added

arielleleon commented:
After meeting with @saskiad , I fixed / added the following:

- using coupled_fov_index
- removed scanimage_resolution for the provided fov_scale_factor
- default power for fov set to PERCENT by default

saskiad commented on examples/multiplane_ophys_session.py:
```diff
@@ -0,0 +1,212 @@
+                    fov_size_unit=SizeUnit.UM,
+                    magnification="10x",
+                    fov_scale_factor=0.78,
+                    fov_scale_factor_unit=SizeUnit.UM,
```
fov_scale_factor_unit should be um/pixel, correct?

arielleleon replied:
Good catch - this is fixed now

saskiad reviewed (changes requested):
Minor thing with the `fov_scale_factor_unit`
